server/statsPage.js

```diff
@@ -58,16 +58,59 @@ const getData = async () => {
 module.exports = function(app, env) {
   app.get('/usage_stats', async (req, res) => {
     try {
-      const table = (await getData())
+      const data = await getData()
+
+      const subjects = {
+        visits: {total: [], monthly: [], daily: []},
+        txSubmissions: {
+          'successful:total': [],
+          'successful:monthly': [],
+          'successful:daily': [],
+          'unsuccessful:total': [],
+          'unsuccessful:monthly': [],
+          'unsuccessful:daily': [],
+        },
+      }
+
+      data.forEach((item) => {
+        const nameTokens = item[0].split(':')
+        const subject = nameTokens[0]
+
+        let period = nameTokens[1]
+        if (subject !== 'visits') {
+          period = `${period}:${nameTokens[2]}`
+        }
+
+        subjects[subject][period].push([nameTokens[nameTokens.length - 1], item[1]])
+      })
+
+      const stats = Object.keys(subjects)
         .map(
-          (v) => `
-        <tr>
-          <td>${v[0]}</td>
-          <td>${v[1]}</td>
-        </tr>
-      `
+          (subject) => `
+          <b>${subject}:</b>
```

Comment:
as the data looks like almost column, it would be nice to show visits and (un) successful submisions side by side for visual comparison  as columns as they have similar rows

Reply:
if you prefer structured lists to table, it would be nice to have daily reports collapsable (but show a few most recent)